vsintegration/src/FSharp.Editor/FSharp.Editor.resx

```diff
@@ -218,8 +218,8 @@ Show structure guidelines for F# code;
 Outlining;
 Show outlining and collapsible nodes for F# code;
 Inline hints;
-Display inline type hints (experimental);
-Display inline parameter name hints (experimental);Beer</value>
+Display inline type hints (preview);
+Display inline parameter name hints (preview);Beer</value>
```

Comment:
I know it was there before, but I am a little confused by the "Beer" at the end here